docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -38,95 +77,117 @@ As for the domain URL, Feel free to get creative — just remember that the URL
-- You can safely leave the `Retention setting` as `Create snapshot`.
-- Under `Availability` we recommend the default value of `Low (one AZ)` for most circumstances.
+![Elastic Beanstalk Networking configuration](images/EBNetworkingConfig.png)
 
-![Elastic Beanstalk Database Settings](images/EBDatabaseSettings.png)
+After configuring the zones where the Load Balancer will exist and the ones that the application will live, click `Save` at the bottom of the page. 
+
+### 2.3 Final step and deploy
+
+Now go to the Capacity section and click `Edit`
+![Elastic Beanstalk Networking configuration](images/EBCapacity.png)
+
+The only change you need to do here is to reduce the number of Instances from 4 (the default number) to 1, as we still haven't created a centralized database where Metabase will save all of its configurations and will be using only the embedded H2 database which lives __inside__ the Metabase container and [is *not recommended* for production workloads](https://www.metabase.com/docs/latest/operations-guide/configuring-application-database.html) as there will be no way to backup and maintain that database. **When your instance is restarted for any reason you'll lose all your Metabase data**. If you are just doing a quick trial of Metabase that may be okay but otherwise you would like to start [creating your database engine in RDS separately](creating-RDS-database-on-AWS.html) or deploy one a separate server.
+
+![Elastic Beanstalk Networking configuration](images/EBCapacityModified.png)
+
+Now click on `Save` at the bottom of the page and you can now click on `Create App` at the end of the Configuration page to start creating the environment.
+
+## Step 3 - Wait for your environment to start
+
+This can take a little while depending on AWS. It’s not uncommon to see this take 10-15 minutes, so feel free to do something else and come back to check on it. What's happening here is each part of the environment is being provisioned with AWS's infrastructure automation functionality named CloudFormation (so you can see the detailed progress for the creation of your environment if you open CloudFormation in another tab).
+
+When it's all done you should see something like this:
+
+![ebcomplete](images/EBComplete.png)
+
+To see your new Metabase instance, simply click on the link under your environment name in the top-left (it will end with `.elasticbeanstalk.com`)
+
+Now that you’ve installed Metabase, it’s time to [set it up and connect it to your database](../setting-up-metabase.md).
 
-Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
+# Additional Options
```

Comment:
I see this is the original term, but how about changing it to something like _Optional extras_?